docs/guides/data_viewers/tableau.md

```diff
@@ -29,35 +28,100 @@ CREATE VIEW h2oai AS (
 );
 ```
 
-
 Note that you should use full path names to local files so that they can be found from inside Tableau.
 Also note that you will need to use a version of the driver that is compatible (i.e., from the same release) 
-as the database format you are using.
-
-# Connecting Tableau Desktop to DuckDB
-
-Tableau provides documentation on how to [install a JDBC driver](https://help.tableau.com/current/pro/desktop/en-us/jdbc_tableau.htm) for Tableau to use.
-Note that Tableau needs to be restarted any time you add or modify drivers.
+as the database format used by the DuckDB tool (e.g., Python module, command line) that was used to create the file.
+
+# Installing the JDBC Driver
+
+Tableau provides documentation on how to [install a JDBC driver](https://help.tableau.com/current/pro/desktop/en-us/jdbc_tableau.htm) 
+for Tableau to use.
+For now, we recommend using the latest bleeding edge JDBC driver (0.8.2) as a number of fixes have been made 
+for time compatibility.
+Note that Tableau (both Desktop and Server versions) need to be restarted any time you add or modify drivers.
+
+## Driver Links
+
+The links here are for a recent version of the JDBC driver that is compatible with Tableau.
+If you wish to connect to a database file,
+you will need to make sure the file was created with a file-compatible version of DuckDB.
+Also, check that there is only one version of the driver installed as there are multiple filenames in use.
+
+* MacOS Desktop.
+  * [Download](https://github.com/duckdb/duckdb/suites/14840102996/artifacts/845094935) the DuckDB JDBC driver. This will a file called `duckdb_jdbc-osx-universal.jar`.
```

Comment:
```suggestion
  * [Download](https://github.com/duckdb/duckdb/suites/14840102996/artifacts/845094935) the DuckDB JDBC driver. This will be a file called `duckdb_jdbc-osx-universal.jar`.
```